### ISSUE TITLE
Fictional E - Commerce Sales Analysis

### ISSUE THREAD

**Issue by mariam7084** [abhisheks008/ML-Crate#580]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Fictional E - Commerce Sales Analysis
:red_circle: **Aim** : Peform EDA
:red_circle: **Dataset** : https://www.kaggle.com/datasets/hassaneskikri/fictional-e-commerce-sales-data
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID (If not, then put NA) :
- Approach for this Project :
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by nkhanna94** [comment]
Full name : Niharika Khanna
GitHub Profile Link :[ nkhanna94](https://github.com/nkhanna94)
Participant ID (If not, then put NA) : NA
Approach for this Project : To analyze fictional e-commerce sales data, I'll start with a detailed EDA to understand data patterns and relationships. After cleaning and preprocessing the data, I'll build predictive models using 3-4 different algorithms and compare these models to determine the best fit based on accuracy scores.
What is your participant role? SSOC S3

**Comment by abhisheks008** [comment]
One issue at a time @nkhanna94

**Comment by nkhanna94** [comment]
Hey I completed the previous issue, please assign this to me now.

**Comment by abhisheks008** [comment]
> Hey I completed the previous issue, please assign this to me now.

Please complete your previous issue.

**Comment by aryan0931** [comment]
Name: Aryan Yadav
github: https://github.com/aryan0931
Participant id:NA
Approach for this project: I will perform Exploratory Data Analysis (EDA) on the provided dataset using techniques such as data cleaning, categorical analysis, outlier detection, and visualization (including correlation matrices, heat maps, and more). This process will involve generating summary statistics and conducting feature engineering to prepare the data for machine learning.
For the machine learning analysis, I will utilize algorithms like Linear Regression, Decision Trees, XGBoost, and K-Nearest Neighbors (KNN) to identify the best-performing model. The tools and libraries used for this analysis will include Pandas, NumPy, Matplotlib, Scikit-Learn, and XGBoost.
Participant role: SSOC S3

**Comment by abhisheks008** [comment]
Implement 5-6 models for this project.
Assigned @aryan0931
